packages/workspace/src/tasks-runner/run-command.ts

```diff
@@ -428,7 +428,7 @@ function addTasksForProjectDependencyConfig(
         }
       }
     }
-  } else {
+  } else if (project.data.targets[dependencyConfig.target]) {
```

Comment:
Use the `projectHasTarget` function that does some safety checks.

Reply:
Yup, will change out.